Rozdział 10/klikaczpro.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1 id="heading">Hello world!</h1>
+    
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+    
+    <script>
+    
+   var offset = 0;
+   var direction ='right';
+   var clicks = 50;
```

Comment:
to w sumie nie są kliki tylko prędkość poruszania się headingu, nazwa jest myląca